packages/webapp/src/components/Drawer/index.tsx

```diff
@@ -29,63 +47,69 @@ const Drawer = ({
   classes = {
     modal: '',
     drawer: '',
-    backdrop: '',
-    header: '',
-    content: '',
+    drawerBackdrop: '',
+    drawerHeader: '',
+    drawerContent: '',
+    drawerContainer: '',
   },
   fullHeight,
   responsiveModal = true,
-}) => {
+}: DrawerProps) => {
   const theme = useTheme();
   const isDesktop = useMediaQuery(theme.breakpoints.up('sm'));
 
+  if (!isOpen) {
+    return null;
+  }
+
   return isDesktop && responsiveModal ? (
-    isOpen && (
-      <ModalComponent
-        className={classes.modal}
-        title={title}
-        titleClassName={styles.title}
-        dismissModal={onClose}
-        buttonGroup={buttonGroup}
-      >
-        <div className={styles.modalContent}>{children}</div>
-      </ModalComponent>
-    )
+    <ModalComponent
+      className={classes.modal}
+      title={title}
+      titleClassName={styles.title}
+      dismissModal={onClose}
+      buttonGroup={buttonGroup}
+    >
+      <div className={styles.modalContent}>{children}</div>
+    </ModalComponent>
   ) : (
     <>
       <div
-        className={clsx(styles.backdrop, isOpen ? styles.openC : '', classes.drawerBackdrop)}
```

Comment:
The click away and filtered overlay behaviour of the drawer were both not working because the correct classnames had become jumbled and the stylesheet styles were no longer applied.